Makefile

```diff
@@ -46,9 +46,11 @@ run: setup-cfg-dir
 	docker run -it --rm \
 		-v $(PWD):/shared \
 		-v /dev/iotlab:/dev/iotlab \
+		-v /dev/ttyCN:/dev/ttyCN \
 		-v /tmp/cfg_dir:/var/local/config \
 		-p $(PORT):8080 \
 		-p $(SERIAL_PORT):20000 \
+		--name "gateway_test" \
```

Comment:
Yeah I think we shouldn't force docker container names, esp. if you re-do a docker run with the same name it will error. I removed 'gateway-test' name in 8c7f540475d736384891f793e30256533a0fd2b7 when deleting the docker-run